flake8 tests shows failures

### Description

pytest shows a lot `FAILED conftest.py::flake-8::FLAKE8 - AttributeError: 'Application' object has no attribute 'parse_preliminary_op.`

#### How to reproduce

pytest 

#### Screenshots

If applicable, add screenshots to help explain your problem.

### Versions

- Operating System : linux, mambaforge

--- a/pytest.ini
+++ b/pytest.ini
@@ -1,0 +1,4 @@
+[pytest]
+log_format = %(asctime)s %(levelname)s %(message)s
+log_date_format = %Y-%m-%d %H:%M:%S
+timeout = 60

